app/helpers/dashboard_helper.rb

```diff
@@ -1,4 +1,8 @@
 module DashboardHelper
+  def add_to_dashboard(widget)
+    button_to(_("Add to dashboard"), widgets_path(:widget=>widget), :method => :post)
+  end
```

Comment:
This isn't yet - its for allowing following PRs to easily add widgets to the dashboard